okta-hosted-login/src/Navbar.jsx

```diff
@@ -13,13 +13,72 @@
 import { useOktaAuth } from '@okta/okta-react';
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Container, Icon, Image, Menu } from 'semantic-ui-react';
+import { Container, Icon, Image, Menu, Modal } from 'semantic-ui-react';
```

Comment:
Longer term, I'd love to see semantic-ui-react pulled out entirely, though that may be irrelevant with upcoming sample changes.

For this particular change, I'm fine with it, because this isn't something I expect devs to want to "keep" in their application after it is working.